plugins/techdocs-backend/src/service/router.ts

```diff
@@ -99,6 +105,8 @@ export async function createRouter(
   const router = Router();
   const { publisher, config, logger, discovery } = options;
   const catalogClient = new CatalogClient({ discoveryApi: discovery });
+  const docsBuildStrategy =
+    options.docsBuildStrategy ?? new DefaultDocsBuildStrategy(config);
```

Comment:
if `fromConfig` static method gets implemented (based on input from @backstage/maintainers) this needs to be updated to something like the below too.

```ts
 const docsBuildStrategy =
    options.docsBuildStrategy ?? DefaultDocsBuildStrategy.fromConfig(config);
```